aspnetcore/mvc/views/working-with-forms.md

```diff
@@ -640,7 +640,7 @@ The generated HTML:
 
 The Select Tag Helper  will automatically generate the [multiple = "multiple"](https://w3c.github.io/html-reference/select.html)  attribute if the property specified in the `asp-for` attribute is an `IEnumerable`. For example, given the following model:
 
-[!code-csharp[](../../mvc/views/working-with-forms/sample/final/ViewModels/CountryViewModelIEnumerable.cs?highlight=6)]
+[!code-csharp[](../../mvc/views/working-with-forms/sample/final/ViewModels/CountryViewModelIEnumerable.cs?highlight=8)]
```

Comment:
Fixes the highlighting:

![image](https://github.com/dotnet/AspNetCore.Docs/assets/28659384/7f1cbaa1-aaa9-49a5-b555-4d80045cb86c)